WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/OrderPaymentSection.swift

```diff
@@ -62,6 +64,28 @@ struct OrderPaymentSection: View {
             .padding()
         }
     }
+
+    @ViewBuilder private var feesRow: some View {
+        if viewModel.shouldShowFees {
+            TitleAndValueRow(title: Localization.feesTotal, value: .content(viewModel.feesTotal), selectionStyle: .highlight) {
+                saveFeeClosure(nil)
+            }
+        } else {
+            Button(Localization.addFees) {
+                let testFeeLine = OrderFeeLine(feeID: 0,
+                                               name: "Fee",
+                                               taxClass: "",
+                                               taxStatus: .none,
+                                               total: "10",
+                                               totalTax: "",
+                                               taxes: [],
+                                               attributes: [])
+                saveFeeClosure(testFeeLine)
```

Comment:
I'm assuming this is temporary right? maybe we should add some annotation to it.